osbs/build/build_request.py

```diff
@@ -562,6 +562,7 @@ def render_add_labels_in_dockerfile(self):
             'vendor': self.spec.vendor,
             'authoritative-source-url': self.spec.authoritative_registry,
             'distribution-scope': self.spec.distribution_scope,
+            'release': self.spec.release,
```

Comment:
@dzickusrh, this key.

I had commented on the commit itself before you created the PR. I believe that's why my comment was no longer attached a specific code line. Anyways, let's track the discussion in Tim's comment now.